packages/examples/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.29.0]
+### Changed
+- **BREAKING:** Improve request function types ([#1014](https://github.com/MetaMask/snaps-monorepo/pull/1014))
+- **BREAKING:** Change checksum calculation -- shasum values updated in snap manifests ([#1128](https://github.com/MetaMask/snaps-monorepo/pull/1128))
+- Standardise logging across all packages ([#1164](https://github.com/MetaMask/snaps-monorepo/pull/1164))
```

Comment:
This is an internal change, and doesn't seem important to mention in the changelog.
```suggestion
```